### ISSUE TITLE
new function : merging packages

### ISSUE THREAD

**Issue by jimcasaer** [inbo/camtraptor#201]
Quite often we want to merge the exports of several research projects (more and more for multi-site research), all using Camtrap-DP as export format, mostly all resulting from agouti.
Though manually this can be done by using several times read_camtrap_dp and subsequently merging the different resulting csv-files, I was wondering if it would not be better to add a function to camtraptor that automatically 

- [ ] merges different data packages resulting from read_camtrap_dp
- [ ] adds a column to the deployments.csv using the project_name as variable
- [ ] adds a column to the deployments.csv using the project location / study site 
- [ ] the function name could be "join_exports" or something similar but this I leave up to Peter :-)

**Comment by peterdesmet** [comment]
I agree with the need of this functionality and that it is best supported by a function in Camtrap DP. The implementation details are to be discussed, but I'd prefer to tackle this after https://github.com/inbo/camtraptor/milestone/3

**Comment by damianooldoni** [comment]
A logic request, definitely. I agree with @peterdesmet: it's better to add this functionality after the big refactoring.

**Comment by damianooldoni** [comment]
Agreed with @peterdesmet to add this functionality to [v1](https://github.com/inbo/camtraptor/milestone/3) and to give it a higher priority.

**Comment by peterdesmet** [comment]
_Suggested in camtraptor July 2023 coding sprint_

- Call this function `merge()`
- Limit it to two packages `merge(package, package2)` to avoid complex operations, testing and error handling. Users can add additional packages using pipe: `package1 %>% merge(package2) %>% merge(package3)`

**Comment by PietrH** [comment]
> A package that is the result of a merge() https://github.com/inbo/camtraptor/issues/201 should combine its metadata (not sure how)

From #254
